Let's call it "displayDidRefresh" everywhere consistently

smfr opened the pull request:
#### 62b307c7fef7c479de44aadd7c3ba6a6f4a5586f
<pre>
Let&apos;s call it &quot;displayDidRefresh&quot; everywhere consistently
<a href="https://bugs.webkit.org/show_bug.cgi?id=252713">https://bugs.webkit.org/show_bug.cgi?id=252713</a>
rdar://105755550

Reviewed by Myles C. Maxfield.

Use &quot;displayDidRefresh&quot; consistently in WebKit, rather than &quot;displayWasRefreshed&quot;.

Rename one private function on DisplayRefreshMonitorManager to disambiguate
internal calls from the monitor.

* Source/WebCore/platform/graphics/DisplayRefreshMonitor.cpp:
(WebCore::DisplayRefreshMonitor::displayDidRefresh):
* Source/WebCore/platform/graphics/DisplayRefreshMonitorManager.cpp:
(WebCore::DisplayRefreshMonitorManager::displayMonitorDisplayDidRefresh):
(WebCore::DisplayRefreshMonitorManager::displayDidRefresh):
(WebCore::DisplayRefreshMonitorManager::displayWasUpdated): Deleted.
* Source/WebCore/platform/graphics/DisplayRefreshMonitorManager.h:
* Source/WebKit/UIProcess/mac/DisplayLink.cpp:
(WebKit::DisplayLink::displayLinkCallback):
(WebKit::DisplayLink::notifyObserversDisplayDidRefresh):
(WebKit::DisplayLink::notifyObserversDisplayWasRefreshed): Deleted.
* Source/WebKit/UIProcess/mac/DisplayLink.h:
* Source/WebKit/UIProcess/mac/DisplayLinkProcessProxyClient.cpp:
(WebKit::DisplayLinkProcessProxyClient::displayLinkFired):
* Source/WebKit/WebProcess/WebPage/EventDispatcher.cpp:
(WebKit::EventDispatcher::notifyScrollingTreesDisplayDidRefresh):
(WebKit::EventDispatcher::displayDidRefresh):
(WebKit::EventDispatcher::startDisplayDidRefreshCallbacks):
(WebKit::EventDispatcher::stopDisplayDidRefreshCallbacks):
(WebKit::EventDispatcher::notifyScrollingTreesDisplayWasRefreshed): Deleted.
(WebKit::EventDispatcher::displayWasRefreshed): Deleted.
(WebKit::EventDispatcher::startDisplayWasRefreshedCallbacks): Deleted.
(WebKit::EventDispatcher::stopDisplayWasRefreshedCallbacks): Deleted.
* Source/WebKit/WebProcess/WebPage/EventDispatcher.h:
* Source/WebKit/WebProcess/WebPage/EventDispatcher.messages.in:
* Source/WebKit/WebProcess/WebPage/MomentumEventDispatcher.cpp:
(WebKit::MomentumEventDispatcher::startDisplayLink):
(WebKit::MomentumEventDispatcher::stopDisplayLink):
(WebKit::MomentumEventDispatcher::displayDidRefresh):
(WebKit::MomentumEventDispatcher::displayWasRefreshed): Deleted.
* Source/WebKit/WebProcess/WebPage/MomentumEventDispatcher.h:
* Source/WebKit/WebProcess/WebProcess.cpp:
(WebKit::WebProcess::displayDidRefresh):
(WebKit::WebProcess::displayWasRefreshed): Deleted.
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/WebProcess.messages.in:

Canonical link: <a href="https://commits.webkit.org/260684@main">https://commits.webkit.org/260684@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3433f600c48cfa1a8357e5c32d1a2eeeb968fff7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108966 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18045 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41789 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/496 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118256 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/112848 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19501 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9341 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101197 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114722 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14628 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97854 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/42748 "Found 1 new test failure: webgl/2.0.y/conformance2/vertex_arrays/vertex-array-object.html (failure)") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96600 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29495 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84490 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10828 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30844 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11575 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7785 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16963 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50440 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7373 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13172 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

litherum commented on Source/WebCore/platform/graphics/DisplayRefreshMonitor.cpp:
```diff
@@ -223,7 +223,7 @@ void DisplayRefreshMonitor::displayDidRefresh(const DisplayUpdate& displayUpdate
     }
 
-    DisplayRefreshMonitorManager::sharedManager().displayDidRefresh(*this);
+    DisplayRefreshMonitorManager::sharedManager().displayMonitorDisplayDidRefresh(*this);
```
displaymonitordisplay?

smfr replied:
dispayMonitor-displayDidRefresh. Considered `monitorDisplayDidRefresh` but there's possible confusion between the computer-display noun "monitor" and the verb "to monitor".

smfr committed:
Let's call it "displayDidRefresh" everywhere consistently
https://bugs.webkit.org/show_bug.cgi?id=252713
rdar://105755550

Reviewed by Myles C. Maxfield.

Use "displayDidRefresh" consistently in WebKit, rather than "displayWasRefreshed".

Rename one private function on DisplayRefreshMonitorManager to disambiguate
internal calls from the monitor.

* Source/WebCore/platform/graphics/DisplayRefreshMonitor.cpp:
(WebCore::DisplayRefreshMonitor::displayDidRefresh):
* Source/WebCore/platform/graphics/DisplayRefreshMonitorManager.cpp:
(WebCore::DisplayRefreshMonitorManager::displayMonitorDisplayDidRefresh):
(WebCore::DisplayRefreshMonitorManager::displayDidRefresh):
(WebCore::DisplayRefreshMonitorManager::displayWasUpdated): Deleted.
* Source/WebCore/platform/graphics/DisplayRefreshMonitorManager.h:
* Source/WebKit/UIProcess/mac/DisplayLink.cpp:
(WebKit::DisplayLink::displayLinkCallback):
(WebKit::DisplayLink::notifyObserversDisplayDidRefresh):
(WebKit::DisplayLink::notifyObserversDisplayWasRefreshed): Deleted.
* Source/WebKit/UIProcess/mac/DisplayLink.h:
* Source/WebKit/UIProcess/mac/DisplayLinkProcessProxyClient.cpp:
(WebKit::DisplayLinkProcessProxyClient::displayLinkFired):
* Source/WebKit/WebProcess/WebPage/EventDispatcher.cpp:
(WebKit::EventDispatcher::notifyScrollingTreesDisplayDidRefresh):
(WebKit::EventDispatcher::displayDidRefresh):
(WebKit::EventDispatcher::startDisplayDidRefreshCallbacks):
(WebKit::EventDispatcher::stopDisplayDidRefreshCallbacks):
(WebKit::EventDispatcher::notifyScrollingTreesDisplayWasRefreshed): Deleted.
(WebKit::EventDispatcher::displayWasRefreshed): Deleted.
(WebKit::EventDispatcher::startDisplayWasRefreshedCallbacks): Deleted.
(WebKit::EventDispatcher::stopDisplayWasRefreshedCallbacks): Deleted.
* Source/WebKit/WebProcess/WebPage/EventDispatcher.h:
* Source/WebKit/WebProcess/WebPage/EventDispatcher.messages.in:
* Source/WebKit/WebProcess/WebPage/MomentumEventDispatcher.cpp:
(WebKit::MomentumEventDispatcher::startDisplayLink):
(WebKit::MomentumEventDispatcher::stopDisplayLink):
(WebKit::MomentumEventDispatcher::displayDidRefresh):
(WebKit::MomentumEventDispatcher::displayWasRefreshed): Deleted.
* Source/WebKit/WebProcess/WebPage/MomentumEventDispatcher.h:
* Source/WebKit/WebProcess/WebProcess.cpp:
(WebKit::WebProcess::displayDidRefresh):
(WebKit::WebProcess::displayWasRefreshed): Deleted.
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/WebProcess.messages.in:

Canonical link: https://commits.webkit.org/260684@main
